README.md

```diff
@@ -1,14 +1,20 @@
 # Twenty Twenty-One
 
-Contributors: wordpressdotorg  
-Requires at least: 5.3  
-Tested up to: 5.5  
-Requires PHP: 5.6  
-License: GPLv2 or later  
-License URI: http://www.gnu.org/licenses/gpl-2.0.html  
+Contributors: wordpressdotorg
+Requires at least: 5.3
+Tested up to: 5.5
+Requires PHP: 5.6
+License: GPLv2 or later
+License URI: http://www.gnu.org/licenses/gpl-2.0.html
 
 ## Description
 
+Twenty Twenty-One is a blank canvas for your ideas - it makes the block editor your best brush.
+
+With new block patterns, which allow you to create a beautiful layout in a matter of seconds, this theme’s soft colors and eye-catching—yet timeless—design will let your work shine.
```

Comment:
The dashes here are also incorrect — it should be:

> eye-catching — yet timeless — design

Reply:
Looks like the issue here is actually GitHub's monospace font, copying it into the editor shows the em dashes correctly.

Reply:
Swapped, good catch on this :+1: